pet-store-react-hooks/src/Home/Home.tsx

```diff
@@ -1,29 +1,32 @@
-import { useCallback, useEffect, useState } from 'react';
+import { useCallback, useEffect, useRef, useState } from 'react';
 
 import { getAllPetsAsync, getPetKindsAsync } from '~infrastructure/api';
-import type { Pet } from '~infrastructure/api-types';
+import type { PetListItem } from '~infrastructure/api-types';
 import { ErrorMessage } from '~infrastructure/components/errorMessage/ErrorMessage';
 import { LoadingIndicator } from '~infrastructure/components/loadingIndicator/LoadingIndicator';
 
 import { PetsTable } from './PetsTable';
 
-import './home.css';
+import './Home.css';
```

Comment:
Git and windows have failed you here. When you rename a file and the only change is in capitalization - git on windows does not count it as a change. As a result everything works fine on your machine but the rename of the files is not tracked on git and other people who pull that commit will have a broken build.

Search online how to fix it.

Reply:
Had to go to the ,get directory and under [core] had to change -> ignorecase = false

Reply:
Now for some reason in the repository the are two home folders - Home and home. One of which should be deleted

Reply:
Move the styles to the correct directory and remove the incorrect one. Test your app after.

Reply:
Deleted the incorrect directory and tested everything.